priv/perf/apps/load_test/lib/test_runner.ex

```diff
@@ -23,35 +23,59 @@ defmodule LoadTest.TestRunner do
   - transactions per seconds
   - period in seconds
 
-  Optional paramters:
-  - Percentile. You can pass percentile as the fourth parameter. By default `mean` value is used.
-
   You can also modify values for tests by providing `TEST_CONFIG_PATH` env variable, it should contain
   the path to json file. For example:
 
     TEST_CONFIG_PATH=./my_file mix run -e "LoadTest.TestRunner.run()" -- "deposits" "1" "5" 90
 
   It fetches all configuration params from env vars.
   """
+
+  alias LoadTest.Service.Metrics
   alias LoadTest.TestRunner.Config
 
+  @circleci_tag "env:perf_circleci"
+
   def run() do
     case Config.parse() do
-      {:ok, {runner_module, config, property}} -> run_test(runner_module, config, property)
+      {:ok, {runner_module, config}} -> run_test(runner_module, config)
       :ok -> :ok
     end
   end
 
-  defp run_test(runner_module, config, property) do
-    result = Chaperon.run_load_test(runner_module, print_results: true, config: config)
+  defp run_test(runner_module, config) do
+    start_datetime = DateTime.utc_now()
+
+    maybe_add_custom_tag(start_datetime)
 
-    # / 1 is to convert result to float (mean is float, percentiles are integers)[O
-    case result.metrics["error_rate"][property] / 1 do
-      0.0 ->
+    Chaperon.run_load_test(runner_module, print_results: true, config: config)
+
+    end_datetime = DateTime.utc_now()
+
+    case Metrics.assert_metrics(start_datetime, end_datetime) do
+      :ok ->
         System.halt(0)
 
-      _ ->
+      {:error, errors} ->
+        # credo:disable-for-next-line
+        IO.inspect("errors: #{inspect(errors)}")
         System.halt(1)
     end
   end
+
+  defp maybe_add_custom_tag(start_date) do
```

Comment:
custom tags are only added during circle ci runs so test runs can be distinguished

Reply:
This goes well with my comment: https://github.com/omgnetwork/elixir-omg/pull/1765#issuecomment-729644723

Are dashboards and monitors able to distinguish by this parameter?

Reply:
yes, tests are distinguished by env tag. dashboards show data for a specific tag. To show a different env, you change the env template parameter:
![2020-11-18_16-37](https://user-images.githubusercontent.com/6567687/99537338-98423b80-29bc-11eb-9550-2384255476ee.png)

For circle ci, custom tags are added which contain current timestamp so two circle ci runs won't intersect

Reply:
Awesome!